src/main/java/pl/wrapper/parking/pwrResponseHandler/dto/ParkingResponse.java

```diff
@@ -13,4 +15,11 @@ public record ParkingResponse(
         String symbol,
         @Nullable LocalTime openingHours,
         @Nullable LocalTime closingHours,
-        Address address) {}
+        Address address) {
+
+    @JsonIgnore
+    public boolean isOpened() {
+        LocalTime now = LocalTime.now();
+        return now.isAfter(openingHours) && now.isBefore(closingHours);
```

Comment:
This will throw nullPointer - the opening and closing hours are annotated with @Nullable